inc/classes/optimization/CSS/class-minify.php

```diff
@@ -18,43 +18,41 @@ class Minify extends Abstract_CSS_Optimization {
 	 * @since 3.1
 	 * @author Remy Perona
 	 *
+	 * @param string $html HTML content.
 	 * @return string
 	 */
-	public function optimize() {
-		$nodes = $this->find( 'link[href*=".css"]' );
+	public function optimize( $html ) {
+		$styles = $this->find( '<link\s+([^>]+[\s\'\"])?href\s*=\s*[\'\"]\s*?([^\'\"]+\.css(?:\?[^\'\"]*)?)\s*?[\'\"]([^>]+)?\/?>', $html );
```

Comment:
No need to escape double quotes: `[\s\'\"]`, `[\'\"]`, `[^\'\"]` => `[\s"\']`, `["\']`, `[^"\']`.